src/router/router.js

```diff
@@ -177,9 +178,22 @@ export const navigate = async function () {
         route.transition = route.transition(previousRoute, route)
       }
 
+      if (navigatingBack === false && route.options && route.options.keepAlive === true) {
```

Comment:
before this block I would recommend creating a variable:
```js
let viewManager = cacheMap.get(route.hash)`
```

The if statement you would do:
```js
if(navigatingBack === false && ....) {
    if(viewManager !== undefined) {
       viewManager.createEntry()
    } else {
       viewManager = new ViewManager()
       cacheMap.set(route.hash, viewManager)
    }
}
```

this allows us to not have to use cacheMap.get in the rest of the function.